### ISSUE TITLE
Failed to restart network.service: Unit network.service not found

### ISSUE THREAD

**Issue by ehartsock** [hashicorp/vagrant#10272]
### Vagrant version
Vagrant 2.1.5

### Host operating system
Fedora release 29 (Twenty Nine)

### Guest operating system
fedora/beta-29-cloud-base

### Vagrantfile
```# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.require_version ">= 2.0.0"
Vagrant.configure(2) do |config|

config.vm.provision "shell", inline: "dnf install nfs-utils rpcbind @development-tools @ansible-node redhat-rpm-config gcc-c++ ruby ruby-devel rubygems -y; gem install ohai"

config.ssh.forward_agent = true

config.vm.define "", autostart: false do |baseimage|
    baseimage.vm.box = "fedora/beta-29-cloud-base"
    baseimage.vm.hostname = "baseimage.ourdomain.com"
    baseimage.vm.network "private_network", :ip => '10.221.122.104'
    baseimage.vm.provider "libvirt" do |vm|
      vm.memory = 2048
      vm.cpus = 2
      vm.driver = "kvm"
      vm.nic_model_type = "e1000"
    end

    config.vm.synced_folder '.', '/vagrant', disabled: true
    config.ssh.username = 'root'
    config.ssh.insert_key = false

    config.vm.provision "ansible" do |ansible|
      ansible.groups = {
      }
      ansible.playbook = "playbooks/vagrant/main.yml"
      ansible.inventory_path = "inventory/development"
      ansible.extra_vars = {
        vagrant_libvirt: true
      }
      # ansible.verbose = 'vvv'
    end
  end
end
```

### Debug output
https://gist.github.com/ehartsock/74e45c02704d16cd6c73312d1c0ab0fa

### Expected behavior
```vagrant up``` and standard provisioning via Ansible

### Actual behavior
Fails with error:

Redirecting to /bin/systemctl restart network.service
Failed to restart network.service: Unit network.service not found.

### Steps to reproduce
1. Use Vagrant 2.1.5 and fedora/beta-29-cloud-base
2. vagrant up

### Note
This also reproducible with Vagrant 2.1.5 and fedora/beta-29-cloud-base on macOS Mojave.

**Comment by jdoss** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1624068#c6 

I think this is an issue with the backport of the fix to 2.1.5 on the vagrant-2.1.5-1.fc30.noarch RPM.

**Comment by chrisroberts** [comment]
Closing as this looks to be addressed in link above. Cheers!
